### PR TITLE
[GLUTEN-1754][CH] Support aggregate functions with multiple arguments

### PR DESCRIPTION
**Pull request by lgbo-ustc**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

Fixes: #1754

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

unit tests

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala [hunk 1]
```diff
@@ -78,8 +78,6 @@ object CHExpressionUtil {
     CORR -> new ValidatorUtil(new DefaultBlackList),
     FIRST -> new ValidatorUtil(new DefaultBlackList),
     LAST -> new ValidatorUtil(new DefaultBlackList),
```
Comment (zzcclp) [inline backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala:80]: remove `,`

**Comment by zzcclp** [comment]
please add the ut for this, for example: covar_pop

**Review by liuneng1994 — APPROVED** [review]
LGTM

**Review by zzcclp — APPROVED** [review]
LGTM

**Comment by liuneng1994** [comment]
LGTM

**Pull request merged**